add-domain

```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
+trackpp.searchmyitem.xyz
```

Comment:
drill -T searchmyitem.xyz
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
xyz.    172800  IN      NS      x.nic.xyz.
xyz.    172800  IN      NS      generationxyz.nic.xyz.
xyz.    172800  IN      NS      z.nic.xyz.
xyz.    172800  IN      NS      y.nic.xyz.
xyz.    3600    IN      SOA     ns0.centralnic.net. hostmaster.centralnic.net. 3000620592 900 1800 6048000 3600